pmda/contacts.py

```diff
@@ -0,0 +1,235 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+"""
+Contact analysis tools
+"""
+from __future__ import absolute_import, division
+
+import MDAnalysis as mda
+from MDAnalysis.lib.distances import distance_array
+import numpy as np
+
+from .parallel import ParallelAnalysisBase
+
+
+def soft_cut_q(r, r0, beta=5.0, lambda_constant=1.8):
```

Comment:
Do we have to duplicate all this code, can't we import it from `MDAnalysis.analysis.contacts`?